packages/ethash/src/index.ts

```diff
@@ -122,8 +119,8 @@ export default class Ethash {
     return keccak256(Buffer.concat(this.cache))
   }
 
-  headerHash(header: BlockHeader) {
-    return rlphash(header.slice(0, -2))
+  headerHash(rawHeader: Buffer[]) {
```

Comment:
It was a misunderstanding on my part of what the input was when I transcribed it to TS. 

It didn't complain because I set BlockHeader to type any since block v3 still wasn't on npm.